### DISCUSSION TITLE
Issue on mobile devices when screen rotate is allowed

### DISCUSSION THREAD

**Discussion by Brave-Game-Studio** [hiukim/mind-ar-js#168] [General]
Hi, I love this library, it's really good, but I am having one issue I'm struggling to get past. 
When the phone rotates (with screen mode set to auto rotate), it struggles to identify the target image - it's fine if autorotate isn't operating e.g. it just stays in portrait. 
Thoughts?  I've tried playing with the image, rotating it and creating new target files, but so far no joy.

**Comment by shabarito** [comment]
Hello.
This solved to me. But the image will be scanned again.


window.onresize = function () {
document.querySelector('a-scene').systems["mindar-image-system"].pause()
document.querySelector('video').remove();
Array.from(document.querySelectorAll('[mindar-image-target]')).forEach((element,index) => {
element.components['mindar-image-target'].updateWorldMatrix(null, )
});
document.querySelector('a-scene').systems["mindar-image-system"]._startVideo()
}

**Reply by Brave-Game-Studio** [reply]
I partly resolved the issue by putting a much higher quality image to identify, meaning an image that had more identification points, I'll note this code as a backup solution.  Thank you
